Thanks for your contribution! Could you fix the flake 8 issue `/tests/test_swagger-tester.py:72:1: E302 expected 2 blank lines, found 1` so we can merge ?

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,8 +1,4 @@
 language: python
-env:
-- TOXENV=py34
-- TOXENV=py27
-- TOXENV=pypy
 python:
     - "3.4"
     - "2.7"

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,6 +3,10 @@ env:
 - TOXENV=py34
 - TOXENV=py27
 - TOXENV=pypy
+python:
+    - "3.4"
+    - "2.7"
+    - pypy-5.4.1
 install:
   pip install -U tox flake8
 before_script:

--- a/tests/test_swagger-tester.py
+++ b/tests/test_swagger-tester.py
@@ -69,6 +69,7 @@ def test_swagger_test_specify_app_url_use_example():
                  use_example=True)
     conn.terminate()
 
+
 def test_issue_51_fixed():
     conn = ConnexionProcess()
     conn.start()

